### DISCUSSION TITLE
CoinbaseWallet doesn't trigger the app on iOS

### DISCUSSION THREAD

**Discussion by luismmdev** [WalletConnect/walletconnect-monorepo#2961] [Developer Support] [answered]
Hi there!

On `iOS`, when selecting the `Coinbase Wallet` in the `Web3Modal`, it does not trigger the app, but instead displays the following:

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/6247440/019872e9-6c00-4eec-8b80-8786bab3feac)

I have the `Coinbase Wallet` application installed. Other wallets, such as `metamask` or `binance`, work as expected, launching the app if is already installed:

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/6247440/a387cd48-1962-4762-8084-b4dfcf88d2b3)

**Comment by crypblizz8** [comment]
@0xAsimetriq to look into

**Comment by xzilja** [comment, marked as answer]
By default modal only handles Coinbase as an extension in the browser, as they don't use WalletConnect, dapp developer needs to add their own sdk to support Coinbase on mobile, this can be done easily via wagmi connector as described in our docs https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/custom-wallets#custom-wagmi-connectors

**Reply by luismmdev** [reply]
Thanks @0xAsimetriq ! I'll check the docs and try it out.

**Comment by chadyj** [comment]
Hey @luismmdev by the way, which dapp are you having this issue on?

**Reply by luismmdev** [reply]
Hi @chadyj! I'm a developer at [dextools](https://dextools.io/app/en).
